### PR TITLE
Solved issue 2070. Added new file align.scss

### LINKED ISSUE #2070: Align features in homepage to left instead of center
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Description :**

Align the features portion text to the left
**Current behaviour:**

Aligned center
**Expected behaviour:**

Aligned to the left
**Steps to reproduce:**
Go to badgeyay.com
  
**Screenshots**

Screenshot from 2019-07-28 01-44-44

**Would you like to work on the issue?**

No. first-timers only-please

### PR DESCRIPTION
**Pull request by ektaarora3501**
Fixes #2070

I have created a new align.scss file for alignment and added it to all.scss.

### CODE REVIEW

**Comment by gabru-md** [comment]
@ektaarora3501 please take a look at what Kush has mentioned :+1:

**Comment by ektaarora3501** [comment]
@kushthedude  can you please tell more about semantic UI classes and how they are used in the project/

**Comment by gabru-md** [comment]
@ektaarora3501 check the official website of Semantic UI [https://semantic-ui.com/](https://semantic-ui.com/) to get an idea.

**Comment by kushthedude** [comment]
@ektaarora3501 Are you still on it ?

**Comment by ektaarora3501** [comment]
Yes.. Actually not able to implement semantic one ..

**Comment by kushthedude** [comment]
> Yes.. Actually not able to implement semantic one ..

@ektaarora3501 See here https://semantic-ui.com/views/item.html#floated-content, Might help you.

**Comment by ektaarora3501** [comment]
Made required changes.
Created a new division in badge-design.hbs
changed align.scss

File: api/.env.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+BADGEYAY_DEBUG=true
```
Comment (abishekvashok) [inline api/.env.example]: We don't need this

File: frontend/app/templates/components/badge-design.hbs [hunk 2]
```diff
@@ -8,14 +8,16 @@
     </div>
     <div class="column features-list">
       <div class="border-left"></div>
+      <div class="align">
```
Comment (abishekvashok) [inline frontend/app/templates/components/badge-design.hbs]: Please intend the lines inside this div. Better move the align class to the child div.

File: .env.example [hunk 3]
```diff
@@ -1,4 +1,4 @@
-BADGEYAY_DEBUG=true
+BADGEYAY_DEBUG=false
```
Comment (abishekvashok) [inline .env.example]: Please don't change this

**Comment by kushthedude** [comment]
@ektaarora3501 Please try to debug yourself: 
- There is a change for more than 10000 lines which you can also see is unnecessary, Rebase the branch or create new

**Comment by ektaarora3501** [comment]
I  have rebased  it again .. If not  working i will create new one..

**Comment by kushthedude** [comment]
> new

still no good

**Pull request closed without merging**